balancer/controllers/chatgpt.py

```diff
@@ -0,0 +1,80 @@
+from django.http import JsonResponse
+from bs4 import BeautifulSoup
+from nltk.stem import PorterStemmer
+import requests
+import openai
+import tiktoken
+import os
+import json
+
+# remove before production
+from django.views.decorators.csrf import csrf_exempt
+
+# summarize web content extracted from url
+
+
+@csrf_exempt
+def extract_webpage(request):
+    openai.api_key = os.getenv('OPENAI_API_KEY')
+    data = json.loads(request.body)
+    webpage_url = data['webpage_url']
+
+    response = requests.get(webpage_url)
+    soup = BeautifulSoup(response.text, 'html.parser')
+    text_contents = soup.find_all('p')
+    text_contents = [p.get_text() for p in text_contents]
+    text_contents = ' '.join(text_contents)
+
+    stemmer = PorterStemmer()
+    text_contents = text_contents.split()
+    text_contents = [stemmer.stem(word) for word in text_contents]
+    text_contents = ' '.join(text_contents)
+    print(text_contents)
```

Comment:
Is this for debugging? Perhaps use python logger with debug message? Let me know if you need help w/ logging config